examples/imagefilter/main.rs

```diff
@@ -64,8 +64,8 @@ impl sixtyfps::Model for Filters {
         self.0.len()
     }
 
-    fn row_data(&self, row: usize) -> Self::Data {
-        self.0[row].name.clone()
+    fn row_data(&self, row: usize) -> Option<Self::Data> {
+        (row < self.row_count()).then(|| Some(self.0[row].name.clone()))
```

Comment:
Does that compile?

Anyway, you can use .get
```suggestion
        self.0.get().map(|x| x.name.clone())
```